test/setup.strict.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Setup (Strict)
+ * This is the bootstrap code that is run before any tests, utils, mocks in 'strict' mode.
+ */
+require('./setup.common')
+
+jest.spyOn(console, 'log')
```

Comment:
👍 very nicely done

Reply:
I would like to confess that those changes are mine, but this is not the case :) actually, the approach of using spies for console methods was already introduced before and just have been removed from sources for a while